include/rocksdb/options.h

```diff
@@ -1255,6 +1255,14 @@ struct ReadOptions {
   // Default: 0 (don't filter by seqnum, return user keys)
   SequenceNumber iter_start_seqnum;
 
+  // Timestamp of operation. Read should return the latest data visible to the
+  // specified timestamp. All timestamps of the same database must be of the
+  // same length and format. The user is responsible for providing a customized
+  // compare function via Comparator to order <key, timestamp> tuples.
+  // The user-specified timestamp feature is still under active development,
+  // and the API is subject to change.
+  const Slice* timestamp;
```

Comment:
@riversand963 May I ask why its type is `Slice` instead of `uint64_t` ？And what time scale it is ?

Reply:
`Slice` is more general than `uint64_t`, and RocksDB does not assume what data type the application uses to represent timestamp.